scripts/ci-wrapper

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -e
+set -x
+
+sudo ./scripts/bootstrap
+git fetch origin ${1}
+
+# checkout index.yaml and sha256sum folder from asset branch in order to compare sha and merge index
+git checkout origin/${1} ./assets/index.yaml || true
+git checkout origin/${1} ./sha256sum || true
+
+charts=$(./scripts/produce-sha256)
+
+if [[ ! -z $charts ]]; then
+  echo $charts | xargs sh -c ./scripts/ci
```

Comment:
nit: for consistency we should use brackets, and `-n` instead of `! -z`
```suggestion
if [[ -n ${charts} ]]; then
  echo ${charts} | xargs sh -c ./scripts/ci
```